frontend-web/webclient/app/Applications/Jobs/View.tsx

```diff
@@ -827,11 +862,104 @@ function parseUpdatesForAccess(updates: JobUpdate[]): ParsedSshAccess | null {
     return null;
 }
 
+interface InterfaceTarget {
+    rank: number,
+    type: "WEB" | "VNC" | "SHELL";
```

Comment:
Is there any cases with `SHELL` that will be added to the results?

Reply:
The IM2 implementation will never produce a value of `"SHELL"` so it is probably safe to remove from the list here.